hazelcast/include/hazelcast/client/serialization/pimpl/SerializationService.h

```diff
@@ -55,30 +57,62 @@ namespace hazelcast {
 
         namespace serialization {
             namespace pimpl {
+                struct HAZELCAST_API ObjectType {
+                    ObjectType() : typeId(0), factoryId(-1), classId(-1) {}
```

Comment:
It looks like we should not need an empty constructor. 
We may event do this private ?

Reply:
It is actually being used from SerializationService::getObjectType.